Global database and summary sheet.R

```diff
@@ -5,43 +5,53 @@
 #
 ######################################################################################################
 
-# LOAD PACKAGES
+# LOAD PACKAGES ----
 # install.packages("librarian")     #Run if librarian is not already installed
 librarian::shelf(
   ggplot2, cowplot, lubridate, rvest, dplyr, compositions, viridis,
-  tidyverse, countrycode, clipr, sjmisc, awalker89 / openxlsx, EnvStats, 
-  gsheet
+   countrycode, clipr, sjmisc, awalker89 / openxlsx, EnvStats, 
+  gsheet, tidyverse
 )
-
+{
 #
 ##
 ### ********************************************************************************************
-####    CREATE GLOBAL DATABASE WITH ALL RISK SHEETS
+####    CREATE GLOBAL DATABASE WITH ALL RISK SHEETS ----
 ### ********************************************************************************************
 ##
 #
-
+ 
 # Load risk sheets
-healthsheet <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Risk_sheets/healthsheet.csv")
-foodsecurity <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Risk_sheets/foodsecuritysheet.csv")
-debtsheet <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Risk_sheets/debtsheet.csv")
-fragilitysheet <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Risk_sheets/fragilitysheet.csv")
-macrosheet <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Risk_sheets/macrosheet.csv")
-Naturalhazardsheet <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Risk_sheets/Naturalhazards.csv")
-Socioeconomic_sheet <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Risk_sheets/Socioeconomic_sheet.csv")
-acapssheet <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Risk_sheets/acapssheet.csv")
-countrylist <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Indicator_dataset/countrylist.csv")
+healthsheet <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Risk_sheets/healthsheet.csv") %>% dplyr::select(-X)
```

Comment:
Loading the .csvs into GitHub adds an initial index column `X` which these edits delete